### ISSUE TITLE
Problem of deform cnn

### ISSUE THREAD

**Issue by zhxianl** [liuwei16/siamese#1]
Hi, I want to ask you some problems of deform CNN as i am confused a few places for reading your code， Can you help me? Can we talk about it?@liuwei16

**Comment by liuwei16** [comment]
Yes, I have found some problems in my implementation, but recently I have no time to tackle it.
Maybe we can have a discuss. Do you have WeChat?

**Comment by zhxianl** [comment]
Hi, I  recently did not log git.... Do  you have a mailbox, i will contact you by Email.@liuwei16
